### ISSUE TITLE
Not support LDAP login 

### ISSUE THREAD

**Issue by shuxiao9058** [nlamirault/emacs-gitlab#41]

**Comment by drym3r** [comment]
It's the same as non-ldap. I can assure you that it woks correctly without modification.

**Comment by norpol** [comment]
I agree, from GitLab's authentication method there is no distinction between "regular" and LDAP login.
